src/test/kotlin/no/nav/tilleggsstonader/sak/util/SøknadBarnetilsynUtil.kt

```diff
@@ -52,10 +53,12 @@ object SøknadBarnetilsynUtil {
 
     fun lagBarnMedBarnepass(
         type: TypeBarnepass = TypeBarnepass.BARNEHAGE_SFO_AKS,
+        utgifter: Utgifter = Utgifter(harUtgifterTilPass = JaNei.JA, null, null),
```

Comment:
Er det mer interresant å teste med noen verider? For eksempel:
```suggestion
        utgifter: Utgifter = Utgifter(harUtgifterTilPass = JaNei.JA, LocalDate.of(2025,1,1), LocalDate.of(2025,5,31)),
```